BRISK_threshold: 10
BRISK_octaves: 4
BRISK_patternScale: 0.1